Cluster: Add `skip` mode to `lxc cluster restore` command (#15267)
canonical/lxd: doc/api-extensions.md

Reviewer: API extension commit message needs updating with new API extension name.

## `container_bpf_delegation`

Adds new {config:option}`instance-security:security.delegate_bpf`.* group of options in order to support eBPF delegation using BPF Token mechanism. See {ref}`bpf-delegation-token` for more information.

> ## `clustering_evacuate_member_only`

Author: yep sorry

> ## `clustering_evacuate_member_only`

Author: fixed

## `override_snapshot_profiles_on_copy`

This adds a request option to set snapshot's target profile on instance copy to be inherited from target instance.

## `clustering_restore_member_only`

> Adds a `--member-only` flag to the `lxc cluster restore` command, allowing to restore a cluster member's status to `ONLINE` without affecting the state or location of instances. When this flag is set, LXD will neither restart local instances that were stopped during evacuation nor migrate back instances that were evacuated to other nodes.

Reviewer 2: I suggest `--status-only`. `lxc cluster restore <member> --member-only` seems confusing.

> Adds a `--member-only` flag to the `lxc cluster restore` command, allowing to restore a cluster member's status to `ONLINE` without affecting the state or location of instances. When this flag is set, LXD will neither restart local instances that were stopped during evacuation nor migrate back instances that were evacuated to other nodes.

Reviewer 2: ```suggestion
Adds a `--member-only` flag to the `lxc cluster restore` command. When this flag is set, LXD restores a cluster member's status to `ONLINE` without restarting any of its stopped local instances or migrating back instances that were evacuated to other cluster members.
```

Author: fixed
